tutorials/profiler.md

```diff
@@ -86,7 +86,7 @@ From terminal 2, open the visualizer using one of the following commands
 
 ```{.sh}
 # Use a launcher script (Linux and macOS)
-ign_remotery_vis
+/usr/lib/ignition/ignition-common4/ign_remotery_vis
```

Comment:
this path is incorrect on macOS; it installs to `/usr/local`

Reply:
Added instructions in 162832f79647bb04c6489b834472f20f69e09024

Reply:
also, the Ubuntu debian install path is `/usr/lib/x86_64-linux-gnu/ignition/ignition-common4/ign_remotery_vis`

Reply:
well done